bin/publish

```diff
@@ -29,17 +29,12 @@ if [ "$git_description" = "v${VERSION}" ]; then
 
   echo "Pushing to Dockerhub..."
   source_image=conjur-authn-k8s-client:dev
-  # tagging and pushing to dockerhub
```

Comment:
let's bring back this comment

Reply:
why do we need the comment if it's mentioned in the `echo` above?

Reply:
b/c at a glance, `$REGISTRY` could be anything (could be dockerhub or could be our Openshift registry for example). Right now it is `cyberark/conjur-authn-k8s-client:latest` but doesn't tell me it is to dockerhub so IMO it is helpful to add the comment above describing it is pushing the image to dockerhub.

Reply:
but 2 lines above it we `echo` "Pushing to Dockerhub...".

Reply:
oh missed that. we good here then